integration/operator_test.go

```diff
@@ -74,3 +80,57 @@ func TestGetReleaseVersion(t *testing.T) {
 		t.Errorf("release didn't match expected, want %q, got %q", expected, actual)
 	}
 }
+
+func TestInstallChart(t *testing.T) {
+	l, err := micrologger.New(micrologger.Config{})
+	if err != nil {
+		t.Fatalf("could not create logger %v", err)
+	}
+
+	config, err := clientcmd.BuildConfigFromFlags("", harness.DefaultKubeConfig)
+	if err != nil {
+		t.Fatalf("could not create k8s config %v", err)
+	}
+	cs, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		t.Fatalf("could not create k8s client %v", err)
+	}
+
+	hc := helm.Config{
+		Logger:     l,
+		K8sClient:  cs,
+		RestConfig: config,
+	}
+
+	h, err := helm.New(hc)
+	if err != nil {
+		t.Fatalf("could not create helm client %v", err)
+	}
+
+	// integration dir is mounted in /e2e in the test container.
+	tarballPath := filepath.Join("/e2e", "tb-chart.tar.gz")
```

Comment:
What is this about?

Reply:
This test relies on a tarball fixture included in this PR. By default e2e-harness mounts `./integration` on `/e2e` in the test container, so that this kind of files can be made available to tests.